.github/workflows/ci_release.yml

```diff
@@ -29,9 +29,13 @@ jobs:
 
   lint:
     uses: ./.github/workflows/lint.yml
+    with:
+      GO_VERSION: "1.21"
 
   test:
     uses: ./.github/workflows/test.yml
+    with:
+      GO_VERSION: "1.21"
```

Comment:
```suggestion
```

Reply:
I think it's needed for compat reasons as v1.22 can bring additional dependencies not ready to be done 😅